docs/changelog/122074.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 122074
+summary: Delete Alias Write Index
```

Comment:
Please make the summary explain what is the visible impact of this PR for the customer.
Maybe sth like "Fix Delete API with delete_dest_index parameter when the destination index alias is used"?